### PR TITLE
Changes after the meeting with CEOS

### PR DESCRIPTION
**Pull request by m-mohr**
Apply the changes discussed in the CEOS meeting today.

Ready for review @maawoo

### CODE REVIEW

**Review by maawoo — COMMENTED** [review]
Other than the comment related to #10, I don't see any issues!

File: sar/README.md [hunk 1]
```diff
@@ -141,9 +143,10 @@ only additional requirements and mappings to fulfill the CARD4L requirements are
...
-| card4l:gridding_convention             | ✗           | ✓ 4.4       | `GriddingConvention`                                         | string                                                  | **REQUIRED if no reference URL/DOI has been provided as a [link with relation type](#stac-item-links) `gridding-convention`** A brief free text description of the gridding convention used. |
+| card4l:geometric_accuracy_type         | ✗           | ✓ 4.3       | `type` in `GeoCorrAccuracy`                                  | string                                                  | **REQUIRED.** Either `slant-range` or `gtc` (Geocoded Terrain Corrected). |
+| card4l:northern_geometric_accuracy     | ✗           | ✓ 4.3       | `GeoCorrAccuracy`                                            | [Geometric Accuracy Object](#geometric-accuracy-object) | **REQUIRED.** An estimate of the northern or line geometric accuracy. |
+| card4l:eastern_geometric_accuracy      | ✗           | ✓ 4.3       | `GeoCorrAccuracy`                                            | [Geometric Accuracy Object](#geometric-accuracy-object) | **REQUIRED.** An estimate of the eastern or sample geometric accuracy. |
```
[the remark this answers is not in the document]
Comment (m-mohr) [reply sar/README.md:148]: Ah, I forgot about that indeed. Interesting again that it's in the metadata, but in the PDF there's no such requirement. There's only a requirement for the data.

[the remark this answers is not in the document]
Comment (m-mohr) [reply sar/README.md:148]: Added, can you have another look, please @maawoo ?

**Review by maawoo — COMMENTED** [review]
Looks good to me 👍🏻

**Pull request merged**